geonode_mapstore_client/client/js/plugins/MetadataEditor/containers/MetadataUpdateButton.jsx

```diff
@@ -35,11 +37,13 @@ function MetadataUpdateButton({
             })
             .catch((error) => {
                 setExtraErrors(get(error, 'data.extraErrors', {}));
-                setUpdateError(get(error, 'data.message', null));
+                let errorObj = { type: "danger", message: getMessageById(context.messages, 'gnviewer.metadataUpdateError') };
```

Comment:
Could we use always the message from the backend for other status too and fallback on the default one if the message is not available? Not sure the conten of error message for other statuses, is it meaningfull?

`message: get(error, 'data.message', '') || getMessageById(context.messages, 'gnviewer.metadataUpdateError')`

Reply:
That was my intention @allyoucanmap yes.
PLease @dsuren1 follow this adivce.

Reply:
I didn't see any meaningful message for other statuses like 404, 500. For this reason the default static error message is shown

Reply:
@dsuren1 yes, please add this change, thanks